app/routes/explore.js

```diff
@@ -252,6 +252,21 @@ export default class ExploreRoute extends Route {
           }
         ]
       });
+    } else if (params.is_upcoming) {
+      filterOptions.push({
+        and: [
+          {
+            name : 'starts-at',
+            op   : 'lt',
```

Comment:
it should be `gt` here

Reply:
Yes ,I made it wrong while fixing indentation . Thanks